libraries/botbuilder-core/src/skills/botFrameworkClient.ts

```diff
@@ -21,21 +21,12 @@ export interface BotFrameworkClient {
      * @param conversationId A conversation ID to use for the conversation with the skill.
      * @param activity Activity to forward.
      */
-    postActivity:
-        | (<T>(
-              fromBotId: string,
-              toBotId: string,
-              toUrl: string,
-              serviceUrl: string,
-              conversationId: string,
-              activity: Activity
-          ) => Promise<InvokeResponse<T>>)
-        | ((
-              fromBotId: string,
-              toBotId: string,
-              toUrl: string,
-              serviceUrl: string,
-              conversationId: string,
-              activity: Activity
-          ) => Promise<InvokeResponse>);
+    postActivity: <T = any>(

```

Comment:
This is the fix to remove the unnecessary `as` casting [here](https://github.com/microsoft/botbuilder-js/pull/2966/files#diff-6c0e837cf326652d3b36fad7a83a81d94cacee1e8835f2f535e017d7d83876adL269). The type signatures are equivalent.